README.md

```diff
@@ -550,7 +550,25 @@ end
 
 #### Providing fake timing data
 
-If you're writing tests that depend on specific timing values, you can provide canned durations using the `fabricate_durations_for_testing_purposes` method, and Scientist will report these in `Scientist::Observation#duration` and `Scientist::Observation#cpu_time` instead of the actual execution times.
+If you're writing tests that depend on specific timing values, you can provide canned durations using the `fabricate_durations_for_testing_purposes` method. This can be done using either the old version (single value) or the new version (hash-based) to include both duration and cpu_time.
+
+##### Old version (Single Value)
```

Comment:
I'd be fine leaving this section of the doc out -- the API has changed but the old version still works, but we may as well encourage the updated version

Reply:
Hokey doke! I've removed the "Old version" section and rephrased.